Traceback (most recent call last):
  File "/Users/knf792/miniconda3/envs/nlp-course/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/knf792/miniconda3/envs/nlp-course/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/knf792/miniconda3/envs/nlp-course/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/knf792/miniconda3/envs/nlp-course/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/knf792/miniconda3/envs/nlp-course/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/Users/knf792/miniconda3/envs/nlp-course/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/knf792/miniconda3/envs/nlp-course/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/knf792/miniconda3/envs/nlp-course/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import statnlpbook.util as util
util.execute_notebook('load_ohhla.ipynb')
# docs = load_albums(j_live)
# docs = load_all_songs("../data/ohhla/train")
docs = load_all_songs("../data/ohhla/train/www.ohhla.com/anonymous/j_live/")
trainDocs, testDocs = docs[:len(docs)//2], docs[len(docs)//2:] 
train = words(trainDocs)
test = words(testDocs)
" ".join(train[0:35])
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Input [0;32mIn [3][0m, in [0;36m<cell line: 1>[0;34m()[0m
[0;32m----> 1[0m [38;5;28;01mimport[39;00m [38;5;21;01mstatnlpbook[39;00m[38;5;21;01m.[39;00m[38;5;21;01mutil[39;00m [38;5;28;01mas[39;00m [38;5;21;01mutil[39;00m
[1;32m      2[0m util[38;5;241m.[39mexecute_notebook([38;5;124m'[39m[38;5;124mload_ohhla.ipynb[39m[38;5;124m'[39m)
[1;32m      3[0m [38;5;66;03m# docs = load_albums(j_live)[39;00m
[1;32m      4[0m [38;5;66;03m# docs = load_all_songs("../data/ohhla/train")[39;00m

File [0;32m~/gits/nlp-course/nlp-book/stat-nlp-book/chapters/../statnlpbook/util.py:47[0m, in [0;36m<module>[0;34m[0m
[1;32m     43[0m             [38;5;28;01mfor[39;00m head [38;5;129;01min[39;00m lists[[38;5;241m0[39m]:
[1;32m     44[0m                 [38;5;28;01myield[39;00m (head,) [38;5;241m+[39m prev_tuple
[0;32m---> 47[0m [38;5;28;01mimport[39;00m [38;5;21;01mmpld3[39;00m
[1;32m     50[0m [38;5;28;01mdef[39;00m [38;5;21mplot_bar_graph[39m(values, labels, rotation[38;5;241m=[39m[38;5;241m0[39m, align[38;5;241m=[39m[38;5;124m'[39m[38;5;124mcenter[39m[38;5;124m'[39m, use_mpld3[38;5;241m=[39m[38;5;28;01mFalse[39;00m):
[1;32m     51[0m     [38;5;124;03m"""[39;00m
[1;32m     52[0m [38;5;124;03m    Plots a bar graph.[39;00m
[1;32m     53[0m [38;5;124;03m    Args:[39;00m
[0;32m   (...)[0m
[1;32m     61[0m 
[1;32m     62[0m [38;5;124;03m    """[39;00m

[0;31mModuleNotFoundError[0m: No module named 'mpld3'
ModuleNotFoundError: No module named 'mpld3'

